src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -170,7 +157,7 @@ private ExtensionsRunner(Extension extension) throws IOException {
         // save custom settings
         this.customSettings = extension.getSettings();
         // initialize the transport service
-        this.initializeExtensionTransportService(this.getSettings());
+        this.getNetty4Transport.initializeExtensionTransportService(this.getSettings(), null);
```

Comment:
This method can be called  like
```
getNetty4Transport.initializeExtensionTransportService(....

Reply:
Why are we passing `null` here? We should avoid it.

Reply:
> Why are we passing `null` here? We should avoid it.

The `initializeExtensionTransportService` method need to call method `extensionTransportService` from `ExtensionsRunner.java`, so we should have a reference to the object. And it will be `this` instead of `null`.